lib/recurly/invoice.php

```diff
@@ -37,30 +37,44 @@
  * @property Recurly_Stub $redemptions
  * @property Recurly_Adjustment[] $line_items
  * @property Recurly_TransactionList $transactions
- * @property Recurly_String $all_transactions A link to all transactions on the invoice. Only present if there are more than 500 transactions
+ * @property string $all_transactions A link to all transactions on the invoice. Only present if there are more than 500 transactions
  * @property int $subtotal_before_discount_in_cents The total of all adjustments on the invoice before discounts or taxes are applied.
  * @property int $credit_customer_notes Allows merchant to set customer notes on a credit invoice. Will only be rejected if type is set to "charge", otherwise will be ignored if no credit invoice is created.
  */
 class Recurly_Invoice extends Recurly_Resource
 {
   /**
    * Lookup an invoice by its ID
+   *
    * @param string Invoice number
-   * @return Recurly_Invoice invoice
+   * @param Recurly_Client $client The recurly client
+   * @return object Recurly_Invoice
+   * @throws Recurly_Error
    */
   public static function get($invoiceNumber, $client = null) {
     return self::_get(Recurly_Invoice::uriForInvoice($invoiceNumber), $client);
   }
 
   /**
+   *
    * Retrieve the PDF version of this invoice
+   *
+   * @param string $locale
+   * @return string
```

Comment:
I suspect this is actually binary data but maybe string is the right name for it?

Reply:
Right, in the same way that [`pack`](http://php.net/manual/en/function.pack.php) also returns a "string", but it's actually binary data. I could call out that it's actually a binary string if that helps for clarity